### DISCUSSION TITLE
How do I set test data in time series predict_model function?

### DISCUSSION THREAD

**Discussion by Michael-Yan-wun** [pycaret/pycaret#3356] [Q&A] [answered]
Hello, 

In my case, I need to train by training data and set it in set_up function, but I need to forecasting the new test data in predict_model function with original trained model. How can I attain this?

Thanks for helping.

**Comment by ngupta23** [comment, marked as answer]
If you using the time_series module, you can provide `X` while using predict_model to specify future exogenous variables (check out the docstring of this method for details).

**Reply by Michael-Yan-wun** [reply]
I use X put in predict_model function, but i expect it to predict with my test data(11, 12), but i think the predict result and graph shows the training data (17722, 12).

**Reply by Michael-Yan-wun** [reply]
i want to use test data like regression and classification.
